CONTRIBUTING.md

```diff
@@ -133,7 +133,7 @@ builds:
 
 ### Write code
 
-[See guidelines](guidelines.md).
+[See guidelines](docs/guidelines.md).
```

Comment:
I think it's fine to refer to the site page here.
```suggestion
[See guidelines](https://typelevel.org/cats/guidelines.html).
```

Reply:
Done, thank you.